rofeeah_ayeni/alpha_challenges/the_basics.md

```diff
@@ -133,25 +133,28 @@ end
 
 * Examine the above code and imagine that you run it.
 * What do you expect the outcome will be? Explain your answer.
+**Answer 9:**
+It would spit out all the names in the array. Since it is a while loop set to loop until 4, the code will add 1 to x everytime until it gets to 4.
 
-**Asnwer 9:**
-
-## Question 11
+## Question 10
 
 * Write a ruby method called `names` that returns an array with two names as strings inside of it.
 
-**Answer 11:**
+**Answer 10:**
 ```ruby
 def names
-  # write your code in here
+  puts ['Rofeeah', 'Shania']
 end
+return names
 ```
 
-## Question 12
+## Question 11
 
 * Write a ruby method called `caps` that takes a string as an argument and returns that string with all capital letters. (Hint: use the format from question 11 to write your method.)
 
-**Answer 12:**
+**Answer 11:**
 ```ruby
-# write your code here
+def caps(string)
+	puts string.upcase!
```

Comment:
You might want to fix your tabbing here - it looks like it's a bit much 😄 

You can change your settings in Sublime to do it for you automatically.